styles.css

```diff
@@ -113,3 +107,29 @@ main {
 .text-decor-none{
     text-decoration: none;
 }
+
+.card-img-horizontal {
+    min-width: 13.5rem;
+    max-width: 13.5rem;
+}
```

Comment:
Why have you used same value for both min-width & max-width ?

Reply:
Fixed the width for image